src/cli/option_splitter_spec.js

```diff
@@ -0,0 +1,47 @@
+import OptionSplitter from './option_splitter'
+
+describe('OptionSplitter', function() {
+  it("doesn't split when nothing to split on", function() {
+    const parts = split('../custom/formatter')
+    expect(parts).to.eql(['../custom/formatter'])
+  })
+
+  it('splits relative unix paths', function() {
+    const parts = split('../custom/formatter:../formatter/output.txt')
+    expect(parts).to.eql(['../custom/formatter', '../formatter/output.txt'])
+  })
+
+  it('splits absolute unix paths', function() {
+    const parts = split('/custom/formatter:/formatter/output.txt')
+    expect(parts).to.eql(['/custom/formatter', '/formatter/output.txt'])
+  })
+
+  it('splits absolute windows paths', function() {
+    const parts = split('C:\\custom\\formatter:C:\\formatter\\output.txt')
+    expect(parts).to.eql(['C:\\custom\\formatter', 'C:\\formatter\\output.txt'])
+  })
+
+  it('does not split a single absolute windows paths', function() {
+    const parts = split('C:\\custom\\formatter')
+    expect(parts).to.eql(['C:\\custom\\formatter'])
+  })
+
+  it('splits extensions and modules', function() {
+    const parts = split('ts:typescript')
+    expect(parts).to.eql(['ts', 'typescript'])
+  })
+
+  it('splits extensions and modules with an absolute windows path', function() {
+    const parts = split('ts:C:/typescript')
+    expect(parts).to.eql(['ts', 'C:/typescript'])
+  })
+
+  it('splits more than 2 parts', function() {
+    const parts = split('part1:part2:part3')
+    expect(parts).to.eql(['part1', 'part2', 'part3'])
+  })
+
+  function split(option) {
+    return OptionSplitter.split(option)
+  }
```

Comment:
Thoughts on converting this to something like
```js
// The following is within the describe block for OptionSplitter

const examples = [
  { 
    description: "doesn't split when nothing to split on",
    input: '../custom/formatter',
    output: ['../custom/formatter']  
  }
  //...
]

examples.forEach(({description, input, output}) => {
  it(description, function() {
    expect(OptionSplitter.split(input)).to.eql(output)
  })
})
```

Reply:
I went ahead and changed it. I'm kind of indifferent either way. The individual `it` blocks were already pretty terse, but this way does remove what boilerplate there was and gets to the point of what you're testing.